syntax = "proto3";

package org.couchers.api.communities;

import "google/protobuf/timestamp.proto";

message Point {
  double lat = 1;
  double lng = 2;
}

message Node {
  int32 node_id = 1;
  string name = 2;
  int32 parent_node = 3;
  repeated Point polygon = 4;
  int32 official_cluster = 5;
  google.protobuf.Timestamp created = 6;
}

service Communities {

}